magento/magento2
# #1391: SaveAssetsKeywordsInterface to delete obsolete keywords

### Description (*)

https://github.com/magento/adobe-stock-integration/issues/1391

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1391

### Manual testing scenarios (*)

1. Edit Image details
2. Add / remove new keyword tags
3. Should delete old keyword asset links not specified on the parameters and insert the new ones.

### Questions or comments


### Contribution checklist (*)
